notifications_utils/s3.py

```diff
@@ -13,8 +18,11 @@ def s3upload(
     content_type='binary/octet-stream',
     tags=None,
     metadata=None,
+    access_key=default_access_key_id,
+    secret_key=default_secret_access_key,
 ):
-    _s3 = resource('s3')
```

Comment:
We're always passing in the credentials, but since this is a separate repo used in multiple places I wanted all changes to be backwards-compatible just in case I missed something.

The code here could be cleaner by removing the idea of default creds.

Reply:
Sure, I'll add one